control-plane/rest/openapi-specs/v0_api_spec.yaml

```diff
@@ -953,6 +953,13 @@ paths:
           schema:
             type: string
             format: uuid
+        - in: query
+          name: allowed-hosts
+          required: false
+          schema:
+            type: array
```

Comment:
Do we want to keep them as array at rest? Its one target nqn accessing replica?

Reply:
> Do we want to keep them as array at rest? Its one target nqn accessing replica?

Yes I think so, rest is only in the openapi for testing/debugging, and it's how I'm actually testing the allowed-hosts.

Reply:
Okay